Source/WebCore/style/Styleable.cpp

```diff
@@ -245,18 +245,11 @@ bool Styleable::isRunningAcceleratedTransformAnimation() const
     return false;
 }
 
-bool Styleable::runningAnimationsAreAllAccelerated() const
+bool Styleable::hasRunningAcceleratedAnimations() const
```

Comment:
Maybe `isRunningAnAcceleratedAnimation`?

Reply:
Or `hasAcceleratedAnimation` or `hasActiveAcceleratedAnimation`? Something about the `hasRunning` bit is confusing to me.